backtest.js

```diff
@@ -18,6 +19,8 @@ const max_rows = 600000
 const showReport = true
 const trading_fees = 0.20
 
+console.log(pg_connectionString)
```

Comment:
This could visibly print secret data to the command line, e.g. the postgres user's password.